src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.json.in

```diff
@@ -254,119 +254,119 @@
         "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-arm64.Cross.android-arm64"
```

Comment:
Does the alias need the `{NetVersion}` too?

Reply:
probably need to make sure we add aliases where there weren't before and I need to make sure I understand what is going on with the win runtime packs

Reply:
I don't think we had the netversion in any aliases before, it was just the pack names themselves (as this was for VS authoring)